.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: pnp
```

Comment:
This should make things a lot faster. You can also do zero installs, but that would mean committing a lot of packages (about 100MB) into version control, which I don't think we want